lib/matplotlib/colorbar.py

```diff
@@ -470,9 +472,6 @@ def __init__(self, ax, mappable=None, *, cmap=None,
             linewidths=[0.5 * mpl.rcParams['axes.linewidth']])
         self.ax.add_collection(self.dividers)
 
-        self.locator = None
-        self.minorlocator = None
-        self.formatter = None
```

Comment:
These lines are redundant, as they're also run in `_reset_locator_formatter_scale()` below.